packages/cxl-ui/src/components/cxl-institute-layout.js

```diff
@@ -0,0 +1,66 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlInstituteLayoutStyles from '../styles/cxl-institute-layout-css.js';
+
+import '@vaadin/vaadin-tabs';
+
+class CXLInstituteLayoutElement extends LitElement {
+  static get properties() {
+    return {};
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlInstituteLayoutStyles];
+  }
+
+  render() {
+    return html`
+      <!-- Visible on mobile -->
+      <header id="header" part="header">
+        <a href="/" class="logo">
+          <span class="cxl">C<span>XL</span></span>
+        </a>
+        <h3>Something something</h3>
+      </header>
+
+      <!-- Visible on mobile -->
+      <vaadin-tabs
+        id="tabs"
+        part="tabs"
+        theme="equal-width-tabs cxl-institute-layout-tabs"
+        selected="1"
+      >
+        <vaadin-tab @click="${() => this.changeTab('sidebar')}">Menu</vaadin-tab>
+        <vaadin-tab @click="${() => this.changeTab('content')}">Something here</vaadin-tab>
+      </vaadin-tabs>
+
+      <div part="icon-nav">
+        <slot name="icon-nav"></slot>
+      </div>
+
+      <div id="sidebar" part="sidebar">
```

Comment:
@chanar sidebar was always designated to by `<aside>` element.

Until an exact custom element strategy is decided, make the change.

Reply:
Are you sure? Any reference?

"The <aside> element is used to identify content that is related to the primary content of the webpage, but does not constitute the primary content of the page. Author information, related links, related content, and advertisements are exampes of content that may be found in an aside element."

https://aastudio.fr/Sidebar-and-Aside-are-different.html

I have never seen aside being used in context of eg. <sidebar>sidebar here</sidebar>

Reply:
Excellent article, and it supports my request.

> The aside tag outside a `<section>` or `<article>` tag
> If <aside> is used outside a <section> or an <article> tags - directly within <body> for instance -, it carries a content related to the site such as : tweets from its author, additional navigation, last blog’s posts, last comments, …

Institute app sidebar only carries information related to content area.

Our main nav is already sitting outside of sidebar. Inside, this would indeed violate `<aside>` content principles.